src/main/scala/com/raphtory/core/implementations/pojograph/entities/external/PojoExplodedEdge.scala

```diff
@@ -19,7 +21,7 @@ class PojoExplodedEdge(objectEdge: PojoExEdge, timestamp:Long) extends ExplodedE
   override def getPropertySet(): List[String] =
     objectEdge.getPropertySet()
 
-  override def getPropertyValue(key: String): Option[Any] =
+  override def getPropertyValue[T: ClassTag](key: String): Option[T] =
     objectEdge.getPropertyAt(key,timestamp)
```

Comment:
is the class tag for objectEdge.getPropertyAt inferred automatically?

Reply:
I think it had been but I think intellij suggested having classtags for accessor-type methods so I put it in. I don't feel too strongly either way though :)